.codecov.yml

```diff
@@ -7,6 +7,8 @@ coverage:
     - "./mocks/*"
     - "./app/internal/identity/yoti*"
     - "./app/internal/telemetry"
+    - "./app/internal/page/fixtures.go"
+    - "./app/internal/page/testing_start.go"
```

Comment:
I don't see the value in testing these as `fixtures` doesn't really do much (and people actually use this page) and if `testing_start` broke we'd see it in the cypress tests

Reply:
Cool, makes sense